components/hint-text/src/index.js

```diff
@@ -28,6 +28,19 @@ const StyledHint = styled('span')({
   color: `${COLOUR.GREY_1}`,
 });
 
+/**
+ *
+ * ### Usage
+ *
+ *
+ * Simple
+ * ```js
+ * <HintText hintText="example">Example</HintText>
```

Comment:
Doesn't need the undefined `hintText` property? Not for simple usage tho? Maybe I am missing something, haven't seen that prop before and it is not documented or listed on the components propTypes